server/controllers/book.js

```diff
@@ -0,0 +1,131 @@
+var OrderBook = module.exports;
```

Comment:
So I agree with what you said about just getting this working for now and working on the more sophisticated/speedy approach later - but we definitely need to take a look at designing a system that runs the book in memory. We can keep a copy in the db, that sort of gets async writes when book updates happen - but in general order matching, and book data should happen directly in memory, yes?